### PR TITLE
add chrome images to test-private-ui release branch images

### PR DESCRIPTION
**Pull request by memodi**
Follow up to #50370, adding chrome images to test-private-ui other release branch images

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12.yaml [hunk 1]
```diff
@@ -50,7 +50,9 @@ images:
 - dockerfile_literal: |
     FROM bin AS builder
     FROM registry.ci.openshift.org/ci/tests-private-base:4.12
-    RUN mkdir -p /go/src/github.com/openshift
+    RUN wget https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm && \
```
Comment (kuiwang02) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12.yaml:53]: @memodi what you add is tests-private-newpr, not tests-private-ui. please correct it.

**Comment by kuiwang02** [comment]
I am not sure if you need it on master, 4.16 and 4.17. if you need it, please also add it in these files.

**Comment by memodi** [comment]
> I am not sure if you need it on master, 4.16 and 4.17. if you need it, please also add it in these files.

master, 4.16, 4.17 already has this change.

**Comment by kuiwang02** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.14-images

**Comment by kuiwang02** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.12-images

**Comment by kuiwang02** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.13-images

**Comment by kuiwang02** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.15-images

**Comment by kuiwang02** [comment]
@memodi job `ci/prow/ci-operator-config-metadata ` is failing. you could run 'make update' in your local and then submit the PR.

**Comment by kuiwang02** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.12-images

**Comment by kuiwang02** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.13-images

**Comment by kuiwang02** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.14-images

**Comment by kuiwang02** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.15-images

**Pull request merged**